### PR TITLE
add crystal snippet

### PR DESCRIPTION
**Pull request by tbpgr**
from https://github.com/tbpgr/atom_crystal_snippet_tb/issues/1

### CODE REVIEW

File: snippets/language-crystal.cson [hunk 1]
```diff
@@ -0,0 +1,406 @@
...
+  'Struct':
+    'prefix': 'Struct'
+    'body': 'Struct'
+  'Symbol':
+    'prefix': 'Symbol'
+    'body': 'Symbol'
+  'Time':
+    'prefix': 'Time'
+    'body': 'Time'
+  'Tuple':
```
Comment (keplersj) [inline snippets/language-crystal.cson:342]: I'm not sure snippets are intended for class autocompletion, but for now it will work well.

**Comment by keplersj** [comment]
Refs: tbpgr/atom_crystal_snippet_tb#1

File: snippets/language-crystal.cson (hunk 1 above)
Comment (tbpgr) [inline snippets/language-crystal.cson:342]: Use case

``` crystal
Array(Int32 | String | Char)
[] of Int32
```

If you think not necessary, you can remove it. :)

Comment (keplersj) [inline snippets/language-crystal.cson:342]: I think it's totally necessary right now, but should be replaced with a proper autocomplete provider soon. I don't think snippets are the correct use for classes like that.

**Pull request merged**